src/libs/SidebarUtils.js

```diff
@@ -265,15 +265,15 @@ function getOptionData(report, reportActions, personalDetails, preferredLocale,
     result.isAllowedToComment = !ReportUtils.shouldDisableWriteActions(report);
 
     const hasMultipleParticipants = participantPersonalDetailList.length > 1 || result.isChatRoom || result.isPolicyExpenseChat;
-    const subtitle = ReportUtils.getChatRoomSubtitle(report);
+    const subtitle = Localize.translateIfNeeded(ReportUtils.getChatRoomSubtitle(report));
```

Comment:
Same here, I'm not sure about wrapping these in `translateIfNeeded`, it seems like `translateIfNeeded` will just return the string passed in if it's not an object, is there any case where `getChatRoomSubtitle` or `getPolicyName` will return an object?